tools/debug/debugsymbolviewer.py

```diff
@@ -159,10 +154,44 @@ def parse_file(log_file):
 					stack_addr = int(sub_word,16)
 					# Check if the stack address lies in kernel text address range
 					if (is_text_address(hex(stack_addr))):
+						if (format_print):
+							if (is_app_crash):
```

Comment:
if `is_app_crash` is true, but the printed msg is kernel text range.
Could you check this?

Reply:
Thanks for pointing out, Updated the code